src/server.tsx

```diff
@@ -58,25 +57,28 @@ app.get('/healthcheck', (req: express.Request, res: express.Response) => {
 interface Epic {
     html: string;
     css: string;
+    testName: string;
+    variantName: string;
 }
 
-const fiveMinutes = 60 * 5;
-const [, fetchDefaultEpicContentCached] = cacheAsync(fetchDefaultEpicContent, fiveMinutes);
+const [, fetchConfiguredEpicTestsCached] = cacheAsync(fetchConfiguredEpicTests, 60);
 
 // Return the HTML and CSS from rendering the Epic to static markup
 const buildEpic = async (
     tracking: EpicTracking,
     localisation: EpicLocalisation,
     targeting: EpicTargeting,
 ): Promise<Epic | null> => {
-    const variant = await fetchDefaultEpicContentCached();
+    const tests = await fetchConfiguredEpicTestsCached();
+    const result = findVariant(tests, targeting);
```

Comment:
I always found this function name a bit confusing, because it's finding both the test and the variant, right? Maybe we should rename to `findTest` or `findTestAndVariant`? Looking at the name along I'd expect this to just find a variant within a test.

Reply:
Yeah I know what you mean. I guess the logic is that a variant always belongs to a test, so the test bit is implied. But I think `findTestAndVariant` is super clear, so I'll go with that.